documentation/asciidoc/topics/con_client_connection_details.adoc

```diff
@@ -0,0 +1,41 @@
+[id='client_connection_details-{context}']
+= Client Connection Details
+Before you can connect to {brandname}, you need to retrieve the following pieces of information:
+
+* Service hostname
+* Port
+* Authentication credentials
+* TLS certificate, if you use encryption
+
+[NOTE]
+====
+In client configuration examples, `$SERVICE_HOSTNAME` denotes the hostname that allows access to your {brandname} cluster.
+Client configuration examples also use port `11222`.
+
+You should replace `$SERVICE_HOSTNAME` with the actual hostname for your environment.
+Likewise, if you use a node port or route, you should replace `11222` with the correct port.
```

Comment:
Instead of saying replace 11222 should we use $SERVICE_PORT so that this is more explicit?

I can imagine users copy and pasting `$SERVICE_HOSTNAME:11222`, replacing the var and forgetting to replace 11222 or worse not realise it's necessary as they did not read this section. WDYT?

Reply:
Thanks, @ryanemerson I've changed to `$SERVICE_HOSTNAME:PORT` (maybe it should be `$PORT`) and made it more explicit that users will need to replace these with the actual values.